include/seqan3/range/views/pairwise_combine.hpp

```diff
@@ -219,7 +219,8 @@ class pairwise_combine_view : public std::ranges::view_interface<pairwise_combin
         requires std::ranges::sized_range<underlying_range_type>
     //!\endcond
     {
-        return (std::ranges::size(u_range) * (std::ranges::size(u_range) - 1) / 2);
+        auto size = std::ranges::size(u_range);
```

Comment:
Why is this necessary? std::ranges::size requires (semantically) constant time. Is there anything more to it, than just reducing one additional size call?
```suggestion
        auto const size = std::ranges::size(u_range);
```

Reply:
I can also drop the PR. I found it easier to read `(size * (size - 1)) / 2` and you are correct that size is constant and I at first thought it wasn't, but this change will remove that doubt.